hibernate-core/src/test/java/org/hibernate/orm/test/query/SQLServerNationalizedScalarQueryTest.java

```diff
@@ -31,10 +31,10 @@
 @TestForIssue(jiraKey = "HHH-10183")
 @RequiresDialect(value = SQLServerDialect.class, majorVersion = 10)
```

Comment:
Since version 10 is for SQL Server 2008, you can now drop the major version 10.

```suggestion
@RequiresDialect(value = SQLServerDialect.class)
```